source/general/faq.rst

```diff
@@ -32,3 +32,43 @@ We encourage members to make suggestions! Suggestions do not need to be based of
 you had seen in the show, but they should remain practical. You can make suggestions by
 clicking `here to go to our forums <https://projectkorra.com/forum/forums/suggestions.8/>`_.
 Please ensure that your suggestion is not a duplicate of another already posted!
+
+Where can I download ProjectKorra?
+==================================
+
+The latest release of ProjectKorra is 1.8.9, which works for Minecraft 1.15 and 1.14. You can download it `here <https://projectkorra.com/forum/resources/projectkorra-core.1/>`_.
+
+Does ProjectKorra work for 1.16?
+================================
+
+The latest release, 1.8.9, was designed for Minecraft 1.15 and 1.14.
+If you installed it on your 1.16 server, the plugin would mostly work, but it would break under certain circumstances. Therefore, it's not recommended to use PK 1.8.9 for Minecraft 1.16.
+The next release will provide support for 1.16. The developers are working hard to ensure there are no game-breaking bugs before releasing, but the next update will come very soon and will add blue fire for firebenders, among other improvements.
```

Comment:
I'd omit this, we don't want to include version specific information for the general faq unless its to say what the latest version is and what its compatible with.

Reply:
Ok, my main reason for making this was because people ask about it constantly. If we could just point them here, we wouldn't have to say the same things over and over. Will this be outdated soon? Yes, but it could help us in the meantime.

Reply:
I'm saying that you should exclude talking about how we are going to add blue fire and other improvements, as well as make it more general following a similar style to the comment I made below.